azure/firewall-policy/versions.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
-  required_version = ">= 1.3.0"
+  required_version = ">= 1.5.0"
 
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "~> 3.22"
+      version = "~> 3.0"
```

Comment:
If we are expanding support for `firewall-policy` to work on azurerm `3.0`, would it also not make sense to have `firewall` on `3.0` also? 

I've tested this on azurerm `3.0` and the plan works so it should work.

I'm in two minds about this though because a part of me feels that we should also be ensuring that are modules only work with the latest version to ensure that we're keeping up to date with the latest standards and because we're almost certainly only going to exclusively test on the latest version. What are your thoughts on this?

Reply:
All other modules are set to 3.0

Reply:
Discussed.  Modules will be set to the version they are initially tested with going forward.